### PR TITLE
Adding documentation for HTTPS

### PR DESCRIPTION
**Pull request by nikhiljindal**
Adding documentation for HTTPS ingresses

cc @csbell @G-Harmon

### CODE REVIEW

**Comment by nikhiljindal** [comment]
Had pushed the commit with wrong author :)
Fixed it now

File: examples/zone-printer/https.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Refer to documentation
+[here](https://github.com/kubernetes/ingress-gce/blob/master/README.md#backend-https)
+for more details.
+
+## Blocking HTTP
+
+While a single cluster ingress also supports blocking HTTP traffic all together,
+kubemci does not support it yet.
+
+https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/issues/142 has
```
Comment (G-Harmon) [inline examples/zone-printer/https.md]: "#142 is the feature request for this" ?

File: examples/zone-printer/https.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+### Caveat
+
+When using a kubernetes secret to specify the SSL cert, kubemci can pick any
+cluster to read the desired secret. Hence the user is required to manage the
+secret in all clusters and ensure that they are in sync.
+
+In addition, updating the secret is not yet supported.
+
+https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/issues/141 has
```
Comment (G-Harmon) [inline examples/zone-printer/https.md]: "#141 has more details and includes a workaround" ?

File: examples/zone-printer/https.md (hunk 1 above)
Comment (nikhiljindal) [reply examples/zone-printer/https.md]: updated

File: examples/zone-printer/https.md (hunk 2 above)
Comment (nikhiljindal) [reply examples/zone-printer/https.md]: Updated

**Comment by nikhiljindal** [comment]
Thanks for the review @G-Harmon 
Updated code as per comments.

Also updated ingress-gce documentation to ensure that it has instructions for both pre-shared cert annotation and secret: https://github.com/kubernetes/ingress-gce/pull/144

**Comment by nikhiljindal** [comment]
cc @prameshj  as FYI

**Pull request merged**